Infrastructure/Documentation/SwaggerApiDocumentation.cs

```diff
@@ -55,64 +39,25 @@ private static void IncludeXmlCommentsIfPresent(SwaggerGenOptions c)
                 c.IncludeXmlComments(xmlPath);
         }
 
-        private static void SeperatePerApiVersion(SwaggerGenOptions c)
-        {
-            c.DocInclusionPredicate((docName, apiDesc) =>
-            {
-                var versions = apiDesc.ControllerAttributes()
-                    .OfType<ApiVersionAttribute>()
-                    .SelectMany(attr => attr.Versions).ToList();
-
-                var any = versions.Any(v => $"{v.GetFormattedApiVersion()}" == docName);
-                return any;
-            });
-        }
-
         /// <summary>
         /// Pre-Requisite Controllers must [ApiVersion("x")] on them
         /// </summary>
         /// <param name="app"></param>
         /// <param name="apiName"></param>
         /// <returns></returns>
-        public static List<ApiVersionDescription> ConfigureSwaggerUiPerApiVersion(this IApplicationBuilder app, string apiName)
+        public static List<ApiVersionDescription> ConfigureSwaggerUi(this IApplicationBuilder app, string apiName)
         {
-            var api = app.ApplicationServices.GetService<IApiVersionDescriptionProvider>(); 
+            var api = app.ApplicationServices.GetService<IApiVersionDescriptionProvider>();
             var apiVersions = api.ApiVersionDescriptions.Select(s => s).ToList();
             app.UseSwaggerUI(c =>
             {
-                //Generate swagger docs per api version
-                foreach (var apiVersionDescription in apiVersions)
-                {
-                    //Create a swagger endpoint for each swagger version
-                    c.SwaggerEndpoint($"{apiVersionDescription.GetFormattedApiVersion()}/swagger.json",
-                        $"{apiName} {apiVersionDescription.GetFormattedApiVersion()}");
-                }
+                //Create a swagger endpoint for each swagger version
+                c.SwaggerEndpoint($"v{1}/swagger.json", $"{apiName} {1}");

```

Comment:
Is there a reason we interpolate 1 here? Or could we have:

```suggestion
                c.SwaggerEndpoint($"v1/swagger.json", $"{apiName} 1");
```